scalameta/contrib/src/main/scala/scala/meta/contrib/ScaladocParser.scala

```diff
@@ -0,0 +1,120 @@
+package scala.meta.contrib
+
+import DocToken._
+import scala.meta.Syntax
+import scala.meta.tokens.Token.Comment
+
+object ScaladocParser {
+
+  /**
+    * Parses a scaladoc [[Comment]].
+    */
+  def parseScaladoc(comment: Comment): Seq[DocToken] = {
+
+    val trimmedComment: String = comment.show[Syntax].trim
+
+    require(
+      requirement = trimmedComment.startsWith("/*") && trimmedComment.endsWith("*/"),
+      message = "Input comment is not a Scaladoc"
+    )
+
+    val scaladocSymbols = Seq('/', '*', ' ')
+
+    val lineSeparedDocString: Seq[String] =
+      trimmedComment
+        .split(Array('\n', '\r')) // Splits the Scaladoc in lines
```

Comment:
That part took a while for me to come up with :) there may be a better/more-efficient way to implement it, but it does the trick and performance is not a big issue.

We can break it down into

```scala
AnyChar // any character
val x = AnyChar ~ !"}}}" // any character that is not followed by }}}, see http://www.lihaoyi.com/fastparse/#NegativeLookahead
x.rep // repeat this parser indefinitely
val y = (x.rep ~ AnyChar) // take one extra character because the character followed by }}} is not included in `x`

y.! // "cut" this parser, which means keep the value of this. an uncut parser discards it's output, like the "}}}" parser
```